src/test/tests/unit/launcher.py

```diff
@@ -51,7 +51,7 @@
 
 # Get the launcher command for new versions of VisIt.
 def GetLauncherCommand(args):
-    p = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    p = subprocess.Popen(args, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
`universal_newlines` is the key to avoiding bytes vs string issues.

Reply:
Good to know. That's probably the fix needed in the sim tests (and the sim support in visit_test_main.py)